Hello Microverse Project Pull Request

AdrianScharff opened the pull request:
It is a simple html and css project
Added css file
Added html file
Added Linters
Added .gitignore file with node_modules and test.md inside

AdrianScharf committed:
Add html file

AdrianScharf committed:
Add css file

AlphaNtihinduka reviewed (changes requested):
## Hi @AdrianScharff 🙋‍♂️
Good job working on your Hello Microverse project. You have done incredibly well. 👏  

You still need to implement few on the project but you are almost there!!! :watch: 

TO HIGHLIGHT:
- Proper use of semantic tags ✔️ 
- Your PR is professional ✅
- Well-structured files ✅
- No linter errors ✔️ 

# STATUS: Required Changes :recycle: 

![close](https://user-images.githubusercontent.com/99394374/210771148-619a4265-33b2-4e68-8901-6bb52693d6de.gif)

# Required Changes :recycle: 
- [ ] I have noticed that your readme is not yet professional. According to the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) you should use the [Microverse readme template](https://github.com/microverseinc/readme-template). A README, along with a repository license, citation file, contribution guidelines, and a code of conduct, communicates expectations for your project and helps you manage contributions. Kindly consider using the template provide and customize it to meet the project requirements ☺️.

## Optional suggestions
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

AdrianScharff committed:
Update README.md

AdrianScharff commented:
Readme File modified with the requirements fullfilled

franclobo commented on README.md:
```diff
@@ -1,2 +1,110 @@
-Linters Configuration
+<a name="readme-top"></a>
+<div>
+ <h3><b>Microverse README Template</b></h3>
```
- [ ] For professional documentation please 🙏🏽 change the title according to your project, "Hello Microverse" is a good idea.

LionRouge1 replied:
- [ ] As mentioned by the previous reviewer, Kindly replace "Microverse README Template" by your project name

> * [ ]  For professional documentation please 🙏🏽 change the title according to your project, "Hello Microverse" is a good idea.

franclobo commented on README.md:
```diff
@@ -1,2 +1,110 @@
+<div>
+ <h3><b>Microverse README Template</b></h3>
+</div>
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+- [📖 About the Project](#about-project)
+- [💻 Getting Started](#getting-started)
+ - [Setup](#setup)
+ - [Prerequisites](#prerequisites)
+ - [Install](#install)
+ - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
```
- [ ] For professional documentation the table of contents should look like in the README template, kindly check and fix the issue 👇🏽 

| Expected | Yours |
|---|---|
| ![image](https://github.com/AdrianScharff/Hello-Microverse/assets/58642949/042cf500-750b-4f98-8185-4d5c04655392) | ![image](https://github.com/AdrianScharff/Hello-Microverse/assets/58642949/197e74e7-0883-4ea3-915e-921519fc3924) |


It is because you need a space between lines 6 and 7 👇🏽 

![image](https://github.com/AdrianScharff/Hello-Microverse/assets/58642949/66f6d876-6f6b-4fa3-a3e0-06c46b120455)

LionRouge1 replied:
- [ ] As mentioned by the previous reviewer, Your table of content look messy, kindly make sure your README file looks like the Microverse template 

> * [ ]  For professional documentation the table of contents should look like in the README template, kindly check and fix the issue 👇🏽
> 
> Expected	Yours
> ![image](https://user-images.githubusercontent.com/58642949/242429502-042cf500-750b-4f98-8185-4d5c04655392.png)	![image](https://user-images.githubusercontent.com/58642949/242429560-197e74e7-0883-4ea3-915e-921519fc3924.png)
> It is because you need a space between lines 6 and 7 👇🏽
> 
> ![image](https://user-images.githubusercontent.com/58642949/242430342-66f6d876-6f6b-4fa3-a3e0-06c46b120455.png)

franclobo commented on README.md:
````diff
@@ -1,2 +1,110 @@
+```sh
+ rails server
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- AUTHORS -->
+## 👥 Authors <a name="authors"></a>
+👤 **Author1**
+- GitHub: [@AdrianScharff](https://github.com/AdrianScharff)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/adrián-juárez-17a51b263)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- FUTURE FEATURES -->
+## 🔭 Future Features <a name="future-features"></a>
+- [ ] **Javascript files**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- CONTRIBUTING -->
+## 🤝 Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](../../issues/).
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- SUPPORT -->
+## ⭐ Show your support <a name="support"></a>
+If you like this project rate a star⭐
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+I would like to thank microverse guiding me through this project.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- FAQ (optional) -->
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+- **Is this project work?**
+ - No, it is template project
+- **Is development finished?**
+ - Not yet.
+ <p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LICENSE -->
+## 📝 License <a name="license"></a>
+This project is [MIT](./MIT.md) licensed.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
````
- [ ] You have the same issue in your content as in the menu, please 🙏🏽 add the spaces needed to look like the README template for professional documentation.

| Expected | Yours |
|---|---|
| ![image](https://github.com/AdrianScharff/Hello-Microverse/assets/58642949/12da4761-2bde-4740-9895-3864ee3611ff) | ![image](https://github.com/AdrianScharff/Hello-Microverse/assets/58642949/a2022c49-60e9-46c8-855c-5a838abcd8dc) |

LionRouge1 replied:
- [ ] Same here as pointed earlier :point_up:

franclobo reviewed (changes requested):
Hi @AdrianScharff, 

# ♻️ Required Changes ♻️

![so-close-jim-halpert](https://github.com/AdrianScharff/Hello-Microverse/assets/58642949/8113d8fe-7a6d-4b0b-a755-7f0da1c525bb)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

LionRouge1 reviewed (changes requested):
Hi @AdrianScharff, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![almost](https://media.giphy.com/media/R2m2NzVxQ3pbG/giphy.gif)

## Suggested changes :recycle: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

AdrianScharff committed:
Update README.md

Spaces added according to the corrections made, so that the document looks professional and according to the template provided by Microverse

AdrianScharff commented:
Sorry for the past requested review, I did the changes on the main branch instead of the feature branch, but hopefully now everything is ok 🙏

Shedrack-Sunday reviewed (changes requested):
Hello @AdrianScharff    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there's one adjustment you need to make to move to the next project. 💯

![](https://user-images.githubusercontent.com/80612925/157397398-00904e26-60ec-46cd-9510-240661c250ef.gif)

## Highlights   
- Good use of GitHub features ✔️  
- Kudus making changes suggested previously by the last reviewer 👍 
  
# Required Changes  🌵 

- [x] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you make In line with the requirements 
- [x] The changes you made created a conflict between this branch and your main branch. Which means you can't merge directly unless you fix and get everything in sync. Kindly follow [This Link](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) to see ways to fix this and that will be all to get approved. Well done 👏

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressingill really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

AdrianScharff committed:
Merge branch 'main' into htmlcss-branch

Shedrack-Sunday reviewed (approved):
Hi @AdrianScharff 👋

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

![image](https://i.pinimg.com/originals/3d/d2/07/3dd207237b39dcacd832f94a35428255.gif)


## Highlights
- Good job in resolving the prior conflict ✔️ 
- Kudus to meet the exact project requirements   👍 

## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.
Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
